sensors/triggers/params.go

```diff
@@ -203,52 +225,51 @@ func ResolveParamValue(src *v1alpha1.TriggerParameterSource, events map[string]*
 	case src.Value != nil:
 		// Use the default value set by the user in case the event is missing
 		resultValue = *src.Value
-		return &resultValue, nil
+		return &resultValue, stringType, nil
 	default:
 		// The parameter doesn't have a default value and is referencing a dependency that is
 		// missing in the received events. This is not an error and may happen with || conditions.
-		return nil, nil
+		return nil, errorType, nil
```

Comment:
The original return seems to be incorrect, it should return an error.

Reply:
checkout the comment in lines 230-231

Reply:
If that's the case, can you return `stringType` as that's the original expectation?

Reply:
have changed it